### PR TITLE
Change docker login

### PR DESCRIPTION
**Pull request by marcoabreu**
This PR changes how we publish our Docker cache. This is required for security reasons.

### CODE REVIEW

File: ci/docker_cache.py [hunk 1]
```diff
@@ -217,7 +249,11 @@ def script_name() -> str:
     args = parser.parse_args()
 
     platforms = build_util.get_platforms()
-    return build_save_containers(platforms=platforms, registry=args.docker_registry, load_cache=True)
+    try:
```
Comment (larroy) [inline ci/docker_cache.py]: Can we use with idiom? Python context? you can make it a separate PR

Comment (marcoabreu) [reply ci/docker_cache.py]: Good idea, will do.

**Comment by marcoabreu** [comment]
I wanted to avoid environment variables since these would have to be defined on the Jenkins master level and it gets quite messy.

**Comment by marcoabreu** [comment]
Self-merge due to urgency. Docker cache is disabled right now.
